network-api/networkapi/people/urls.py

```diff
@@ -1,11 +1,2 @@
-from django.conf.urls import url
-
-from networkapi.people.views import (
-    PeopleListView,
-    PersonView,
-)
-
 urlpatterns = [
```

Comment:
Is this file needed at all anymore if the urlpatterns are empty?

Reply:
it is not, but part 2 removes it too